pom.xml

```diff
@@ -207,6 +207,24 @@
             <version>${hadoop.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.20.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
Just for consistency sake and to avoid adding the overhead of managing possibly more CVE's from new deps- 
since junit is already imported in the pom, I suppose we can use that instead of assertj-core ?

also, I see that we have used `awaitility` here. I've generally seen usage of TestUtils for the same functionality https://github.com/confluentinc/ce-kafka/blob/970dc45276ba77f606cd162bfb60fdbdb0f26a02/clients/src/test/java/org/apache/kafka/test/TestUtils.java#L280

https://github.com/confluentinc/kafka-connect-kinesis/blob/770d244822fa04b4270ef5cd9ac0a6de0d862f64/src/test/java/io/confluent/connect/kinesis/integration/BaseConnectorIT.java#L138-L140

Reply:
Thanks. Are test dependencies included in CVE scans? 
I saw `TestUtils.waitForCondition` but having the possibility of using assertions makes the test more concise.
Otherwise, if you want useful information about why a condition was not met, you would need to use the one that has a `conditionDetailsSupplier` argument and duplicate some of the test condition logic.

Reply:
Ah you're right, I doubt CVE's can come from here since it's a test dependency. Thanks @erdody 👍🏽